src/game/editor/popups.cpp

```diff
@@ -1282,6 +1287,14 @@ bool CEditor::PopupMenuFile(void *pContext, CUIRect View)
 		return true;
 	}
 
+	View.HSplitTop(10.0f, &Slot, &View);
+	View.HSplitTop(12.0f, &Slot, &View);
+	if(pEditor->DoButton_MenuItem(&s_EntitiesButton, "Load Entities", 0, &Slot, 0, "Load a image as the editor game entities"))
```

Comment:
"Load entities" looks much more natural
```suggestion
	if(pEditor->DoButton_MenuItem(&s_EntitiesButton, "Load entities", 0, &Slot, 0, "Load an image as the editor's game entities"))
```